README.md

```diff
@@ -15,6 +15,17 @@ Deploy Wazuh platform using Chef cookbooks. Chef recipes are prepared for instal
 
 Each cookbook has its own README.
 
+## Use through Berkshelf
+
+The easiest way to making use of these cookbooks (especiall `wazuh_filebeat` & `wazuh_elastic` until they are published to Supermarket) is by including in your `Berksfile` something like the below:
```

Comment:
Hi @curiositycasualty I noticed a typo here. Did you meant especially?

Regards,
Jose
```suggestion
The easiest way to making use of these cookbooks (especially `wazuh_filebeat` & `wazuh_elastic` until they are published to Supermarket) is by including in your `Berksfile` something like the below:
```